migrations/20201123023013-create-application_notes.ts

```diff
@@ -0,0 +1,48 @@
+import Base from 'db-migrate-base';
+import { promisify } from 'util';
+
+/**
+ * Create the Application Notes table for Mod Dash notes on an organization
+ */
+export async function up(
+  db: Base,
+  callback: Base.CallbackFunction
+): Promise<void> {
+  db.runSql(
+    `
+    CREATE TABLE application_notes
+      (
+        id                           SERIAL,
+        note                         TEXT,
+        created_at                   TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+        updated_at                   TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+        organization_id              INTEGER,
```

Comment:
changing this to NOT NULL because all application notes should have an organization id